udfs/cloudbuild.yaml

```
@@ -14,19 +14,143 @@
 #
 
 steps:
-###########################################################
-# Step 1: Run UDF Unit Tests
-###########################################################
-- name: python:3.8.0-slim
-  entrypoint: 'bash'
-  args: ['udfs/tests/run.sh', '--pip_install_before_run']
-  env: ['SHORT_SHA=$SHORT_SHA']
-###########################################################
-# Step 2: Deploy if all tests pass
-###########################################################
+  ###########################################################
+  # Step 0: Install npm packages
+  ###########################################################
+- name: 'node'
+  id: 'step_0_install_npm_packages'
+  entrypoint: 'npm'
+  args:
+    - install
+  dir: js_libs
+  ###########################################################
+  # Step 1: Build all js libraries for BigQuery UDFs
+  ###########################################################
+- name: 'node'
+  id: 'step_1_build_bq_js_libs'
+  entrypoint: 'npm'
+  args:
+    - run-script
+    - build-all-libs
+  dir: js_libs
+  waitFor:
+    - 'step_0_install_npm_packages'
+  ###########################################################
+  # Step 2: Copy libs to GCS bucket
+  ###########################################################
 - name: gcr.io/google.com/cloudsdktool/cloud-sdk
-  entrypoint: 'bash'
-  args: ['release/build.sh', '$BRANCH_NAME', '$_PR_NUMBER']
+  id: 'step_2_copy_js_to_gcs'
+  entrypoint: 'gsutil'
+  args:
```

speed it up! use `gsutil -m` to parallelize uploads
```suggestion
  args:
    - "-m"
```

---

Good catch! Done